### PR TITLE
Update thoughtbot brand red

### PR DESCRIPTION
**Pull request by tysongach**
Our current brand red color (`#ed3e44`) has a contrast ratio of 3.91:1 (against a background color of pure white). When we use this red as a text color, which we often do, the contrast does not meet WCAG ~~2.0~~ 2.1 guidelines (specifically 1.4.3, Level AA) and poses accessibility issues.

WCAG ~~2.0~~ 2.1 specifies that text should have a contrast ratio of at least 4.5:1 (3:1 for large-scale text, which is at least 18 point or 14 point bold) to meet Level AA.

This updated red (`#e03131`) has a contrast ratio of 4.51:1.

Resources:

- https://www.w3.org/TR/WCAG21/#contrast-minimum
- https://webaim.org/resources/contrastchecker/
- https://usecontrast.com/

**To see the current red versus the new red, you can view the production and the Netlify Deploy Preview:**

- Current red: https://presskit.thoughtbot.com/
- New red: https://deploy-preview-30--thoughtbot-presskit.netlify.com/

---

On a more opinionated note: after looking at the updated red, the current red feels very washed out. The new red carries a bit more richness.

### CODE REVIEW

File: source/index.html.erb [hunk 1]
```diff
@@ -136,11 +136,11 @@
...
         <div class="color-specs">
           <h3>Primary Color</h3>
-          <h4>rgb(237, 62, 68)</h4>
-          <h4>#ed3e44</h4>
+          <h4>rgb(224, 49, 49)</h4>
+          <h4>#e03131</h4>
           <hr>
-          <h4>Pantone® P 48-8u*</h4>
-          <h4>cmyk(0, 99, 91, 0)</h4>
+          <!-- <h4>Pantone® P 48-8u*</h4> -->
```
Comment (tysongach) [inline source/index.html.erb]: We’ll want to update this, if we move forward.

**Comment by setphen** [comment]
The accessibility standards used are from [2008](https://www.w3.org/TR/WCAG20/), which makes me wonder if there needs to be some consideration for monitor gamut changes in the past decade. I think their contrast ratios are great, but the calculations for determining the ratios between two hex codes might need some tweaking if we consider changes to popular color profiles across new devices.

**Comment by almnt** [comment]
100% support this. I’m a low-vision individual and the changes made to the red are definitely easier to read.

**Comment by tysongach** [comment]
@setphen Good stuff to think about for sure! [As we spoke about in Slack](https://thoughtbot.slack.com/archives/C02C7SZUQ/p1528472371000477), probably good to align with the established standard for now.

I also just remembered that [WCAG 2.1 was made a Recommendation this week](https://www.w3.org/blog/2018/06/wcag21-rec/), so I can update this to indicate that it’s compliant with 2.1, which is quite current.

**Comment by samkap** [comment]
I support this as well. Thank you for giving us the comparison!

**Comment by tysongach** [comment]
@MilesRJohnson @LloydBennett I wanted to loop you into this PR, given your efforts over on the [brand guidelines project](https://hub.thoughtbot.com/projects/thoughtbot-brand-guildlines). There’s still a couple of open items to address (see original PR message), but it feels like a good time to pick this back up and potentially mix it in with the rest of the brand updates. I’d love to hear your thoughts and help push this through!

**Comment by tysongach** [comment]
This has been updated:

- Went through the zipped asset pack, updated reds, then re-zipped.
- Reviewed the Pantone color for print using our color chips in the office; wound up sticking with the same color. We may want to tweak it later, but it's hard to do without printing real material tests.

I'm going to plan to merge this soon, unless someone feels otherwise!

**Comment by tysongach** [comment]
I’ve open a series of PRs to update the red on many of our websites:

- https://github.com/thoughtbot/thoughtbot.com/pull/1567
- https://github.com/thoughtbot/maitre-d/pull/525
- https://github.com/thoughtbot/hub/pull/2284
- https://github.com/thoughtbot/constable/pull/584
- https://github.com/thoughtbot/ask/pull/137

**Comment by tysongach** [comment]
Here’s a fun thing: a Contrast Grid of our branding colors: https://contrast-grid.eightshapes.com/?background-colors=&foreground-colors=%23FFFFFF%2C%20White%0D%0A%23e03131%2C%20Primary%0D%0A%2329292c%2C%20Secondary&es-color-form__tile-size=large

**Comment by MilesRJohnson** [comment]
@tysongach this is super cool - thanks for sharing!

**Pull request merged**